projects/admin/src/app/record/detail-view/entities-detail-view/local/local-organisation-detail-view/local-organisation-detail-view.component.html

```diff
@@ -0,0 +1,118 @@
+-->
+<dl class="row mb-0">
+  <!-- NAME -->
+  <ng-container *ngIf="record.name">
+    <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
+      Name
+    </dt>
+    <dd class="col-sm-7 col-md-8 mb-0">
+      {{ record.name }}
+    </dd>
+  </ng-container>
+
+  <!-- SUBORDINATE UNIT -->
+  <ng-container *ngIf="record.subordinate_unit">
+    <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
+      Subordinate unit
+    </dt>
+    <dd class="col-sm-7 col-md-8 mb-0">
+      <ul class="list-unstyled mb-0">
+        <li *ngFor="let name of record.subordinate_unit">{{ name }}</li>
+      </ul>
+    </dd>
+  </ng-container>
+
+  <!-- CONFERENCE PLACE -->
+  <ng-container *ngIf="record.conference_place">
+    <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
+      Conference place
+    </dt>
+    <dd class="col-sm-7 col-md-8 mb-0">
+      {{ record.conference_place }}
+    </dd>
+  </ng-container>
+
+  <!-- CONFERENCE NUMBERING -->
+  <ng-container *ngIf="record.conference_numbering">
+    <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
+      Conference numbering
+    </dt>
+    <dd class="col-sm-7 col-md-8 mb-0">
+      {{ record.conference_numbering }}
+    </dd>
+  </ng-container>
+
+  <!-- CONFERENCE DATE -->
+  <ng-container *ngIf="record.conference_date">
+    <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
+      Conference date
+    </dt>
+    <dd class="col-sm-7 col-md-8 mb-0">
+      {{ record.conference_date }}
+    </dd>
+  </ng-container>
+
+  <!-- CONFERENCE -->
+  <ng-container *ngIf="record.conference">
+    <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
+      Conference
+    </dt>
+    <dd class="col-sm-7 col-md-8 mb-0">
+      <i
+      class="fa fa-circle mr-2"
+      [ngClass]="{'text-success': record.conference, 'text-danger': !record.conference}"
+    ></i>
+    </dd>
+  </ng-container>
+
+  <!-- START DATE -->
+  <ng-container *ngIf="record.start_date">
+    <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
+      Start date
+    </dt>
+    <dd class="col-sm-7 col-md-8 mb-0">
+      {{ record.start_date }}
+    </dd>
+  </ng-container>
+
+  <!-- END DATE -->
+  <ng-container *ngIf="record.end_date">
+    <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
+      End date
+    </dt>
+    <dd class="col-sm-7 col-md-8 mb-0">
+      {{ record.end_date }}
+    </dd>
+  </ng-container>
+
+  <!-- ALTERNATIVE NAMES -->
+  <ng-container *ngIf="record.alternative_names">
+    <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
+      Alternative names
+    </dt>
+    <dd class="col-sm-7 col-md-8 mb-0">
+      <ul class="list-unstyled mb-0">
+        <li *ngFor="let name of record.alternative_names">{{ name }}</li>
+      </ul>
+    </dd>
+  </ng-container>
+</dl>
+
+<!-- GLOBAL FIELDS -->
+<admin-entities-local-global [record]="record"></admin-entities-local-global>
```

Comment:
Personnaly, I would have to use content injection system instead of component specification. It will allow to add a global `entity-wrapper-class` into `<admin-entities-local-global>` component and use the same styling for each subcomponent.
Something like :
```html
<admin-entities-local-global [record]="record">
   <dl>
     <dt translate>my_label_title</dt>
     <dd>my_data</dd>
   <dl>
</admin-entities-local-global>
```